client/app/main/employees/add/emp.dialog.html

```diff
@@ -48,9 +40,7 @@
 
           <md-input-container flex="50">
             <label>Skills</label>
-            <input required name="empSkills" ng-model="employee.skills">
-            <!-- <md-option value="app">Application</md-option>
-                <md-option value="web">Website</md-option> -->
+            <input required name="Skills" ng-model="employee.skills">
             <div ng-messages="employeeForm.skills.$error">
               <div ng-message="required">This is required.</div>
```

Comment:
Make sure that the name that you have for ng-messages is equal to the name of the input, for example
`<input required name="skills" ng-model="employee.skills">
              <div ng-messages="employeeForm.skills.$error">`
In this case you have Skills and skills, those are different